src/utils/fetcher.ts

```diff
@@ -0,0 +1,8 @@
+export default async function fetcher(url: string) {
+  try {
+    const data = await fetch(url).then((res) => res.json())
```

Comment:
Why to use async/await and .then at same line, both serve same purpose :confused:

Reply:
it converts the data returned to serializable json format